src/algorithms/machinelearning/tensorflowpredict.h

```diff
@@ -85,9 +85,9 @@ class TensorflowPredict : public Algorithm {
 
     declareParameter("inputs", "will look for this namespaces in poolIn. Should match the names of the input nodes in the Tensorflow graph", "", inputNamesVector);
     declareParameter("outputs", "will save the tensors on the graph nodes named after `outputs` to the same namespaces in the output pool", "", outputNamesVector);
-    declareParameter("isTraining", "Whether to return the output in training mode (normalized with statistics of the current batch) or in inference mode (normalized with moving statistics)", "{true,false}", false);
-    declareParameter("isTrainingName", "The name if the input reciving isTraining. If empty, it wont be feed ro the network", "", "");
-    declareParameter("squeeze", "Remove singleton dimensions of the inputs tensors. Does not apply to the batch dimension.", "{true,false}", true);
+    declareParameter("isTraining", "run the model in training mode (normalized with statistics of the current batch) or in inference mode (normalized with moving statistics). It only applies to some models", "{true,false}", false);
```

Comment:
"It only applies" --> "This only applies"

Reply:
" or in inference mode" --> "instead of inference mode" ? Because the parameter value is true or false.